From a4af67296588e0c0c6bcdcba5c5b6d6d48f945f4 Mon Sep 17 00:00:00 2001
From: kevintm78 <kevintm78@gmail.com>
Date: Fri, 15 Feb 2019 01:31:43 -0500
Subject: [PATCH] Extended Battery patch-- Credits @theLoneWolf

---
 drivers/battery/max77843_fuelgauge.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/drivers/battery/max77843_fuelgauge.c b/drivers/battery/max77843_fuelgauge.c
index 7a4e4c5..b5363ba 100755
--- a/drivers/battery/max77843_fuelgauge.c
+++ b/drivers/battery/max77843_fuelgauge.c
@@ -357,7 +357,9 @@ static int max77843_fg_read_soc(struct max77843_fuelgauge_data *fuelgauge)
 		return -1;
 	}
 
-	soc = ((data[1] * 100) + (data[0] * 100 / 256)) / 10;
+	// Extended Battery Mod- by TheLoneWolf20
+	//soc = ((data[1] * 100) + (data[0] * 100 / 256)) / 10;
+	soc = max77843_fg_read_vfsoc(fuelgauge);
 	vf_soc = max77843_fg_read_vfsoc(fuelgauge);
 
 #ifdef BATTERY_LOG_MESSAGE
-- 
2.7.4

